daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/crypto/Hash.scala

```diff
@@ -295,4 +307,7 @@ object Hash {
       .iterateOver(parties.toSeq.sorted[String].iterator, parties.size)(_ add _)
       .build
 
+  private val HexaLookupTable =
```

Comment:
This makes me sad, I would have expected that converting to `base16` is something that is not something that you have to reinvent but a quick stackoverflow search does not reveal anything better either https://stackoverflow.com/a/332101/3877993 and it’s probably not worth incurring an extra dependency for so happy to keep this for now.

Reply:
We are indirectly depending on guava. It must be fine to use the encoder from there.

Reply:
:+1: